Sources/MongoSwift/MongoCollection+Write.swift

```diff
@@ -26,6 +26,11 @@ extension MongoCollection {
             // TODO SWIFT-139: include writeErrors and writeConcernErrors from reply in the error
             throw MongoError.commandError(message: toErrorString(error))
         }
+
+        guard isAcknowledged(options?.writeConcern) else {
```

Comment:
I ws going to say "nit: I would probably one-line all these", but I realize we have no stylistic standard for when to use one line vs. two. I usually just kind of do it based on what feels right and how long the line is... any thoughts on how to establish a consistent pattern on that throughout the codebase?

Reply:
I'm not a fan of using one-liners for conditionals. You could make the argument that `guard` is special since we can trust that it will exit the current scope one way or another (`return` or `throw`).

That said, I think they're OK for short lambas (e.g. something that just returns a value and is used with `map`).

I suppose this is something that can be detected with SwiftLint. Does that actually support fixing the code, or can it only alert us when code passes certain checks?

Reply:
https://jira.mongodb.org/browse/SWIFT-217